doc/manual_rest.rst

```diff
@@ -79,10 +80,9 @@ command:
     EXIT STATUS
     ===========
 
-    Exit status is 0 if the command was successful, and non-zero if there was any error.
-
-    Note that some issues such as unreadable or deleted files during backup
-    currently doesn't result in a non-zero error exit status.
+    Exit status is 0 if the command was successful.
+    Exit status is 1 if there was a fatal error (no snapshot created).
+    Exit status is 3 if some source data could not be read (incomplete snapshot created).
```

Comment:
Perhaps this should be in a separate commit as it's unrelated to the copy command?

Reply:
I've split the change into a separate commit.